ramls/holdingsrecordssource.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "A holdings records source",
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": "string",
+      "description": "The system assigned unique ID of the holdings records source; UUID",
```

Comment:
Does it have to be system assigned, what happens if the client provides it on creation?